apps/experiments/decorators.py

```diff
@@ -25,11 +25,14 @@ def decorator(view_func):
         def decorated_view(request, team_slug: str, experiment_id: str, session_id: str):
             request.experiment = get_object_or_404(Experiment, public_id=experiment_id, team=request.team)
             request.experiment_session = get_object_or_404(
-                ExperimentSession, experiment=request.experiment, external_id=session_id, team=request.team
+                ExperimentSession,
+                experiment_id=request.experiment.get_working_version_id(),
```

Comment:
fine to leave this call here but just confirming that this should already be the working version right?

Reply:
Not necessarily in the future, but for now yes. If we have a button that allows the user to chat to a specific version, then this might be the versioned experiment